### PR TITLE
Bootstrap 5

### PR DESCRIPTION
**Pull request by Katrienvh**
OPGELET: Styling bootstrap 5 datepickers verwijderd, dus mag pas gemerged worden als nieuwe datepicker er in zit.

Moet samen gemerged worden met:

- https://github.com/sumocoders/FrameworkCoreBundle/pull/76
- https://github.com/sumocoders/application-skeleton/pull/46

### CODE REVIEW

File: src/sass/_bootstrap-variables.scss [hunk 1]
```diff
@@ -366,6 +366,8 @@ $caret-spacing:               $caret-width * 0.85 !default;
 
 $transition-base:             all 0.2s ease-in-out !default;
 $transition-fade:             opacity 0.15s linear !default;
+// CUSTOM SUMOCODERS
```
Comment (tijsverkoyen) [inline src/sass/_bootstrap-variables.scss:360]: Would/t it be better to follow the code styling that is already used, and write the comment in lowercase. Maybe also explain why this variable needs to exists and where and why it is used.

File: src/sass/_bootstrap-variables.scss [hunk 2]
```diff
@@ -879,6 +880,7 @@ $form-validation-states: (
 $zindex-dropdown:                   1000 !default;
 $zindex-sticky:                     1020 !default;
 $zindex-fixed:                      1030 !default;
+$zindex-backtotop:                  1035 !default; // custom sumocoders
```
Comment (tijsverkoyen) [inline src/sass/_bootstrap-variables.scss:874]: `// custom for sumocoders`

File: src/sass/_bootstrap-variables.scss [hunk 3]
```diff
@@ -1440,5 +1440,24 @@ $kbd-bg:                            $gray-900 !default;
 
 $pre-color:                         null !default;
 
+// CUSTOM VARIABLES
+//
+// Sidebar
+$sidebar-width-open:    235px !default;
```
Comment (tijsverkoyen) [inline src/sass/_bootstrap-variables.scss]: out of curiosity, why are these values absolute values and not calculated based on an more global variable? For instance `5 * $spacer`

File: src/sass/themes/_dark.scss [hunk 4]
```diff
@@ -6,31 +6,27 @@ $red: #b92b0b !default;
 $orange: #b0580e !default;
 $yellow: #c69605 !default;
 $green: #047f12 !default;
-$cyan: #118899 !default;
+$cyan: #189 !default;
```
Comment (tijsverkoyen) [inline src/sass/themes/_dark.scss:9]: Is this according to the codestyles?

Comment (Katrienvh) [reply src/sass/themes/_dark.scss:9]: yes

**Review by tijsverkoyen — APPROVED** [review]

**Pull request merged**